packages/fxa-shared/subscriptions/stripe.ts

```diff
@@ -231,3 +259,12 @@ export const getSubscriptionUpdateEligibility: (
 
   return SubscriptionUpdateEligibility.INVALID;
 };
+
+export const getMinimumAmount = (currency: string): number => {
+  if (STRIPE_MINIMUM_CHARGE_AMOUNTS.hasOwnProperty(currency))
+    return STRIPE_MINIMUM_CHARGE_AMOUNTS[currency];
```

Comment:
nit: I think we usually wrap the if block in `{}` even when there's just one statement.  (Not a convention written down anywhere though. 🤷)

Reply:
Interesting. That should be the ESLint `curly` rule, IIRC.
https://github.com/mozilla/eslint-plugin-fxa/blob/fbcfb9db9881d644d147fa9461a4273884160553/lib/configs/default.js#L21

The ESLint `plugin:fxa/default` config should set this to `'curly': [2, 'all'],` (where `2` is old-style for "error").
I don't see why this would not be enabled via the following configs:
| ESLint CONFIG | EXTENDS
|:----|:-----
| https://github.com/mozilla/fxa/blob/main/_dev/.eslintrc | `"extends": ["plugin:fxa/recommended"],`
| https://github.com/mozilla/fxa/blob/main/packages/fxa-shared/.eslintrc | `"extends": ["plugin:fxa/recommended"],`
| https://github.com/mozilla/eslint-plugin-fxa/blob/fbcfb9db9881d644d147fa9461a4273884160553/lib/configs/recommended.js | `extends: ['plugin:fxa/default', 'prettier']`
| https://github.com/mozilla/eslint-plugin-fxa/blob/fbcfb9db9881d644d147fa9461a4273884160553/lib/configs/default.js | `extends: ['eslint:recommended'],`

And yet, looking at ESLint's `--parse-config` for packages/fxa-shared/subscriptions/stripe.ts, it looks like the `curly` rule is disabled:
```sh
npx eslint --config=_dev/.eslintrc.json --print-config packages/fxa-shared/subscriptions/stripe.ts | jq '.rules.curly' | npx prettier --parser=json
# [0, "all"]
```

So I think the only other candidate is probably the rule is being disabled by **eslint-config-prettier**:
https://github.com/prettier/eslint-config-prettier/blob/7ee41f151c7b17d2a6d5cc1211b3910712eadc1b/index.js#L10

```js
    "curly": 0,
```

Reply:
I'll play around a bit with my `formatOnSave` settings in VSCode. That might be the cause of this.

For now I've manually fixed it.